### PR TITLE
TCK tests for life cycle methods

### PR DESCRIPTION
**Pull request by njr-11**
Add TCK for Insert, Update, and Delete life cycle methods.
Cover both single entity as well as multiple entities in the parameter.
Use an entity with a generated version and confirm it gets updated and is used to make updates.
Confirm that exceptions are raised per the spec requirements.

I also removed two extra `*` characters that I noticed in JavaDoc.

### CODE REVIEW

File: tck/src/main/java/ee/jakarta/tck/data/standalone/persistence/PersistenceEntityTests.java
Comment (mswatosh) [inline tck/src/main/java/ee/jakarta/tck/data/standalone/persistence/PersistenceEntityTests.java:139]:
```suggestion
    @Assertion(id = "133", strategy = "Attempt to insert an entity that already exists in the database and expect EntityExistsException.")
```

File: tck/src/main/java/ee/jakarta/tck/data/standalone/persistence/PersistenceEntityTests.java
Comment (mswatosh) [inline tck/src/main/java/ee/jakarta/tck/data/standalone/persistence/PersistenceEntityTests.java:200]:
```suggestion
assertEquals(3, added.length);

```
Should we verify that added has a length of exactly 3, like is being done below to confirm modified is exactly 2?

**Review by mswatosh — COMMENTED** [review]
Overall the tests look good, just added a few suggestions.
(This still needs approval from a committer)

**Review by jmstephensgit — APPROVED** [review]

**Pull request merged**
